src/actions/reportActions.ts

```diff
@@ -53,30 +53,30 @@ export const submitReport = (report: Report) => async (dispatch: any) => {
  * @param {string} id
  * @param {string} reply
  */
-export const replyReport = (id: string, reply: string) => async (dispatch: any) => {
-  const res = await reportService.replyReport(id, reply)
-  console.log('Action: replyReport response: ', res)
-  return res.data
-}
+export const replyReport =
+  (id: string, reply: string, userType: string) => async (dispatch: any) => {
+    const res = await reportService.replyReport(id, reply, userType)
+    return res.data
+  }
 
 /**
  * @function
  * @desc This function sets arcived status for a report
  * @param {string} id
  * @param {string} archived Note: sent as a string
  */
-export const archiveReport = (id: string, archived: string) => async (dispatch: any) => {
-  const res = await reportService.archiveReport(id, archived)
-  return res.data
-}
+export const archiveReport =
+  (id: string, archived: string, userType: string) => async (dispatch: any) => {
+    const res = await reportService.archiveReport(id, archived, userType)
+    return res.data
+  }
 
 export const fetchReportById = (id: string) => async (dispatch: any) => {
   try {
     const data = await reportService.fetchReportById(id)
-    console.log("report's data", data)
     dispatch({ type: SET_CURRENT_REPORT, data: data })
   } catch (error) {
-    console.log(error)
+    console.error
```

Comment:
```suggestion
    console.error(error)
```
Muotoilin viestin vähän huonosti aamulla :D tarkotin siis tämmöstä console.error funktiota joka tulostaa konsoliin virheen tavallisen viestin sijaan. Se on vähän parempi kuin tuo pelkkä console.log
https://developer.mozilla.org/en-US/docs/Web/API/console/error